uds/message/service_identifiers.py

```
@@ -6,15 +6,15 @@
-    "ALL_REQUEST_SIDS", "ALL_RESPONSE_SIDS",
-    "SERVICES_WITH_SUBFUNCTION",
+    "ALL_REQUEST_SIDS", "ALL_RESPONSE_SIDS", "SidAlias",
+    "SERVICES_WITH_SUBFUNCTION", "SERVICES_WITH_DID", "SERVICES_WITH_RID",
     "RESPONSE_REQUEST_SID_DIFF",
     "RequestSID", "ResponseSID",
     "UnrecognizedSIDWarning",
     "define_service",
 ]
 
-from typing import Tuple
+from typing import Tuple, Union
```

Ruff (0.15.20)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)